library/ceph_volume.py

```diff
@@ -649,7 +649,8 @@ def run_module():
         try:
             report_result = json.loads(out)
         except ValueError:
-            strategy_change = "strategy changed" in out
+            strategy_change = ("strategy changed" in out or
+                               "strategy changed" in err)
```

Comment:
`"strategy changed" in err + out` wdyt ?

Reply:
I prefer it this way, if we have any sort of error keeping ``out`` and ``err`` separately might make debugging easier.